README.md

```diff
@@ -1 +1,13 @@
-# flat-search
\ No newline at end of file
+# flat-search
+
+This is a small project for looki-up a flash according to the options stored in config file.
+
+This cli should be installed to run migrations:
+```
+https://github.com/mattes/migrate
+```
+Note: to run migrations please use the following command:
+```bash
+$GOPATH/bin/migrate -url mysql://[DB_USER_NAME]:[DB_USER_PASSWORD]@tcp\([IP_OR_HOSTNAME]:[PORT]\)/[DB_NAME] -path ./migrations/ up
```

Comment:
@grubastik to make it cool and flexible, please add make file with all needed commands: 

```
make build
make migrate
make run
make run.docker
make docker.stop
make docker.start
etc 
```